app/controllers/friendships_controller.rb

```diff
@@ -0,0 +1,99 @@
+class FriendshipsController < ApplicationController
+  include LolConnections
+  before_action :search_summoner_for_request_action, only: [:edit]
+  before_action :require_user
+  before_action :require_same_user, only: [:edit, :decide]
+  
+ 
+	def new
```

Comment:
This indentation looked good in my editor....I double-checked when making this commit - https://github.com/danfstucky/LeagueSpec/commit/60b313a63b15ac10c4effcb6c16158219ace0d88